_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,339 @@
+---
+layout:       pattern
+title:        Lists
+date:         2015-12-21 00:00:00
+
+categories:   patterns
+tags:
+- atomic
+- pattern
+- lists
+
+slug:         lists
+
+url_styles:     "patterns/_lists.scss"
+
+description: Lists are used to group related content in a number of settings and cases.
+
+info: Lists are used to group related content in a number of settings (e.g. ordered, unordered) and cases (e.g. definitions, links, etc.).
+---
+
+<h3 class="hd-6 example-set-hd">List (using different semantics)</h3>
+<div class="example-set">
+    <ul class="list">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ol class="list">
+        <li class="item">Blueberries</li>
+        <li class="item">Cherries</li>
+        <li class="item">Coconuts</li>
+        <li class="item">Cranberries</li>
+    </ol>
+
+    <div class="list">
+        <p class="item">Figs</p>
+        <p class="item">Oranges</p>
+        <p class="item">Plums</p>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Unstyled Lists</h3>
+<div class="example-set">
+    <ul class="list-unstyled">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Inline List</h3>
+<div class="example-set">
```

Comment:
It might be good to include an example of an inline sequence (but one that is not a ```nav``` )  Just a thought.  I guess, the majority of inline sequences could be considered a navigation element.